harvester/ogm_repositories_config.yaml

```diff
@@ -72,50 +77,46 @@ edu.berkeley:
   metadata_format: gbl1
   filename_regex: |
     .+?/geoblacklight\.json
+  external_url_strategy:
+    name: base_url_and_slug
+    base_url: "https://geodata.lib.berkeley.edu/catalog"
+    gbl1_field: layer_slug_s
 edu.cornell:
   name: "CUGIR, Cornell University"
   metadata_format: gbl1
   filename_regex: |
     .+?/geoblacklight\.json
-edu.nyu:
-  name: "NYU Libraries"
-  metadata_format: gbl1
-  filename_regex: |
-    .+?/geoblacklight\.json
+  external_url_strategy:
+    name: field_value
+    gbl1_field: dc_identifier_s
 edu.princeton.arks:
   name: "Princeton"
   metadata_format: gbl1
   filename_regex: |
     .+?/geoblacklight\.json
+  external_url_strategy:
+    name: base_url_and_slug
+    base_url: "https://maps.princeton.edu/catalog"
+    gbl1_field: layer_slug_s
 edu.columbia:
   name: "Columbia"
   metadata_format: gbl1
   filename_regex: |
     .+?/geoblacklight\.json
-edu.uarizona:
```

Comment:
Yes: Arizona does not have an external URL to point back to.  I should have contextualized this in the PR: some repos, after discussion with stakeholders, were okay for removal because they had few records, old records, no external system, or otherwise did not provide meaningful records.